### ISSUE TITLE
Ventura 13.4 Support?

### ISSUE THREAD

**Issue by walterrowe** [Free-GPGMail/Free-GPGMail#84]
I see GPGsuite is at release 2023.2 but claims it supports up to Ventura 13.3.

Free GPGmail is 2023.1.

I upgraded from Monterey to Ventura 13.4 tonight.

The Free GPGmail plugin won't enable.

**Comment by walterrowe** [comment]
FYI .. GPGsuite GPGMail 7 works with Ventura 13.4.

**Comment by bnavigator** [comment]
AFAICT GPGtools has not released the source code for 2023.2 yet. It's been two months already.

See my comment https://github.com/Free-GPGMail/Free-GPGMail/issues/77#issuecomment-1490295344 though. All we need is the UUID.


```bash
sw_vers -productVersion
```

```bash
defaults read /System/Applications/Mail.app/Contents/Info PluginCompatibilityUUID
```

You might even work around it yourself when you add the UUID to the Info.plist and codesign locally:

```bash
codesign --deep --force --verbose --sign "-" ~/Library/Mail/Bundles/Free-GPGMail_7.mailbundle
```

**Comment by walterrowe** [comment]
Code signing the bundle worked! Thanks!

I also asked for the source link to be posted.

**Comment by bnavigator** [comment]
So what's the UUID I should add?

**Comment by walterrowe** [comment]
```shell
 % defaults read /System/Applications/Mail.app/Contents/Info PluginCompatibilityUUID
281F8A5C-0AF9-4BE6-8B8A-C0CB9C2068BE
```

I ran the exact command you have me below.

```shell
% codesign --deep --force --verbose --sign "-" ~/Library/Mail/Bundles/Free-GPGMail_7.mailbundle
/Users/me/Library/Mail/Bundles/Free-GPGMail_7.mailbundle: replacing existing signature
/Users/me/Library/Mail/Bundles/Free-GPGMail_7.mailbundle: resource fork, Finder information, or similar detritus not allowed
```

After that I could enable the Free-GPGMail_7 plugin.

**Comment by bnavigator** [comment]
But that UUID is already there:

https://github.com/Free-GPGMail/Free-GPGMail/blob/d4b0446c9250941ecffba515759b20503b094ee0/Source/GPGMail_7/Resources/Info.plist#L56-L60

and the released Free-GPGMail 7 2023.1 should have a proper signature, so why didn't it work before your local signature?

**Comment by walterrowe** [comment]
I have no idea. I did a "diff -r" between the Free-GPGMail_7 plugin I downloaded and the copy I dropped into place and codesigned. No differences. Perhaps it places some data somewhere else or registers it with the system somehow?

**Comment by walterrowe** [comment]
Not sure if this helps. It says the signature is timestamped March 19th (your release date) so I don't know what codesign did or where it wrote its changes or how it affected the permissions for Mail.

```shell
% codesign -dv ~/Library/Mail/Bundles/Free-GPGMail_7.mailbundle
Executable=/Users/me/Library/Mail/Bundles/Free-GPGMail_7.mailbundle/Contents/MacOS/Free-GPGMail
Identifier=org.free-gpgmail.gpgmail7
Format=bundle with Mach-O universal (x86_64 arm64)
CodeDirectory v=20400 size=5533 flags=0x0(none) hashes=166+3 location=embedded
Signature size=8981
Timestamp=Mar 19, 2023 at 4:40:17 PM
Info.plist entries=36
TeamIdentifier=F3GFG4ZMDN
Sealed Resources version=2 rules=13 files=194
Internal requirements count=1 size=188
```

**Comment by walterrowe** [comment]
I asked in their support page for the link to the source. No reply. Your step above made Free-GPGMail work for me.

**Comment by fuomag9** [comment]
> I asked in their support page for the link to the source. No reply. Your step above made Free-GPGMail work for me.

They are not replying for BUYING orders either...

**Comment by mattcen** [comment]
Hi folks,

I chased this up a months ago and have just heard back (see below). The latest [source code is now available](https://releases.gpgtools.com/GPG_Suite-2023.2.txz) and [linked from the homepage](https://gpgtools.org/); apparently the only change is the UUID.

I have created PR #85, but looks like there are a bunch of changes (mainly similar to `s/GPGMail/Free-GPGMail/g`) and I wasn't sure of the best way to apply those.
